src/containers/SecretsModal/SecretsModal.js

```diff
@@ -47,6 +47,8 @@ function validateInputs(value, id) {
   return true;
 }
 
+window.HTMLElement.prototype.scrollIntoView = function() {};
```

Comment:
What does this do? A comment above would be useful

Reply:
done 😸

Reply:
We shouldn't be reaching out to native DOM functionality like this, especially not overriding prototypes from the application code.

If we absolutely need this (I'm guessing jsdom doesn't implement scrollIntoView) we should do it from the test code instead.

Reply:
ok, I've moved this to the test code

Reply:
still needs to be removed from here